mingw-w64-clang/PKGBUILD

```diff
@@ -449,29 +466,53 @@ package_mingw-w64-clang-x86_64-llvm(){
   package_llvm
 }
 
-package_mingw-w64-i686-clang(){
+package_mingw-w64-clang-armv7-clang(){
   package_clang
 }
 
-package_mingw-w64-i686-compiler-rt(){
+package_mingw-w64-clang-armv7-compiler-rt(){
   package_compiler-rt
 }
 
-package_mingw-w64-i686-libc++(){
+package_mingw-w64-clang-armv7-libc++(){
   package_libcxx
 }
 
-package_mingw-w64-i686-openmp(){
+package_mingw-w64-clang-armv7-openmp(){
   package_openmp
 }
```

Comment:
Nit: let's remove openmp for i686 and arm to keep it consistent with x86_64 and aarch64.

Reply:
What I really should do is use the template I put together for MINGW-packages (msys2/MINGW-packages#7931)